devices/Pcd8544/Pcd8544.cs

```diff
@@ -0,0 +1,758 @@
+                _controller.Write(backlightPin, PinValue.Low);
+            }
+
+            _controller.OpenPin(_dataCommandPin, PinMode.Output);
+
+            Initialize();
+        }
+
+        private void Initialize()
+        {
+            // Load the font
+            Font5x8 font8 = new Font5x8();
+            foreach (var charac in font8.SupportedChars)
+            {
+                font8.GetCharData((char)charac, out SpanUshort charData);
+                byte[] font8Bytes = new byte[8];
+                for (int i = 0; i < 8; i++)
+                {
+                    font8Bytes[i] = (byte)charData[i];
+                }
+
+                CreateCustomCharacter(charac, font8Bytes);
+            }
+
+            _bias = 4;
+            _temperature = ScreenTemperature.Coefficient0;
+            _contrast = 0x30;
+            _enabled = true;
+            // Extended function, contrast to 0x30, temperature to coef 0, bias to 4, Screen to normal display power on, display to normal mode
+            SpanByte toSend = new byte[] { (byte)(FunctionSet.PowerOn | FunctionSet.ExtendedMode), (byte)(0x80 | _contrast), (byte)ScreenTemperature.Coefficient0, (byte)(0x10 | _bias), (byte)FunctionSet.PowerOn, (byte)PcdDisplayControl.NormalMode };
+            SpiWrite(false, toSend);
+            Clear();
+            Draw();
+        }
+
+        #region properties
+
+        /// <summary>
+        /// Enable the screen
+        /// </summary>
+        public bool Enabled
+        {
+            get => _enabled;
+            set
+            {
+                _enabled = value;
+                byte enab = (byte)(_enabled ? FunctionSet.PowerOn : FunctionSet.PowerOff);
+                SpanByte toSend = new byte[] { enab };
+                SpiWrite(false, toSend);
+            }
+        }
+
+        /// <summary>
+        /// Change the back light from 0.0 to 1.0.
+        /// If a pin is used, the threshold for full light is more then 0.5.
+        /// </summary>
+        public float BacklightBrightness
+        {
+            get => _backlightVal;
+            set
+            {
+                _backlightVal = value > 1 ? 1 : value;
+                _backlightVal = _backlightVal < 0 ? 0 : _backlightVal;
+                if (_pwmBacklight != null)
+                {
+                    _pwmBacklight.DutyCycle = _backlightVal;
+                }
+
+                if (_backlightPin >= 0)
+                {
+                    _controller.Write(_backlightPin, _backlightVal > 0.5 ? PinValue.High : PinValue.Low);
+                }
+            }
+        }
+
+        /// <summary>
+        /// The bias for 0 to 7. Bias represent the voltage applied to the LCD. The highest, the darker the screen will be.
+        /// </summary>
+        public byte Bias
+        {
+            get => _bias;
+            set
+            {
+                _bias = value < 8 ? value : throw new ArgumentException($"Bias can't be more than 7");
```

Comment:
bonus point for all the exceptions: we are removing the text inside when there is nothing specific like in this case.
And we are using the /// intellisence comments instead. So would be really perfect if you can do it for those.